src/main/java/org/gridsuite/ds/server/utils/PropertyType.java

```diff
@@ -10,9 +10,10 @@
  * @author Thang PHAM <quyet-thang.pham at rte-france.com>
  */
 public enum PropertyType {
```

Comment:
Why do you have this enum ?
Why not using https://github.com/powsybl/powsybl-dynawo/blob/bcce6b19c1d42b2eb36cc713357482be94e6dcbc/dynawaltz/src/main/java/com/powsybl/dynawaltz/suppliers/PropertyType.java ?
And ask for some evolutions if needed

Reply:
The one in dynawaltz is a new class and it is not compatible 100% to the one in microservices, especially enum type. At this moment I provide an utility to transform between them for avoid so much of modification in code (in dynay-app, mapping-server, dynamic-simulation-server). Another TS should be created for unify two classes at the right moment.

Reply:
Yes they should enrich the class in powsybl-dynawalz so that it fits to your need in this microservice, that's it ?

Reply:
Not really, for ENUM may be we have to adapt in microservices, on other hand the one of Dynawaltz should provide FLOAT type. I have to see Laurent for details.

Reply:
Ok good